README.md

```diff
@@ -45,7 +45,7 @@ server to become Sticky's production server.
 
 The code in this repository depends on the following software:
 
- - [ansible] >= 2.4
+ - [ansible] >= 2.5
```

Comment:
Do we want a `requirements.txt` or something based on pipenv?

Reply:
We _could_, but in my opinion that's still a bit overkill at the moment.